src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1109,20 +1105,23 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
             // prevent infinite recursion:
             Debug.Assert(!ReferenceEquals(copyFrom, this));
 
+            // The property is responsible for completion of the backing field
+            // NB: when the **field keyword feature** is implemented, it's possible that synthesized field symbols will also be merged or shared between partial property parts.
+            // If we do that then this check should possibly be moved, and asserts adjusted accordingly.
+            _ = BackingField?.GetAttributes();
```

Comment:
@cston This comment and the below assert are likely relevant for the field keyword feature.